src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -65,14 +65,14 @@ public void CanGetGeneralInfoViaAPI()
             try
             {
                 var fullNode = this.apiTestsFixture.stratisPowNode.FullNode;
-                var ApiURI = fullNode.Settings.ApiUri;
+                var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;
```

Comment:
can you please change this variable name to `apiUri`?

Reply:
Program.cs was omitted from the commit. Added.